np11.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+
+def diagonal_array():
+    """
+    Function that create a 3-D array with ones on a diagonal and zeros elsewhere
+    """
+    array = np.zeros((3, 3))
+    np.fill_diagonal(array, 1, wrap=True)
```

Comment:
You can also use numpy.eye
https://numpy.org/devdocs/reference/generated/numpy.eye.html